.github/workflows/ci.yaml

```diff
@@ -92,3 +92,15 @@ jobs:
           file: coverage.xml
           fail_ci_if_error: False
           verbose: True
+          flags: fast-tests
+
+      - name: codecov-schedule
+        if: ${{ github.repository == 'OpenFreeEnergy/openfe'
+                && github.event_name == 'schedule' }}
```

Comment:
schedule is always a bit of a pain because codecov has that annoying thing where it fails if it gets the same commit too many times, would it make sense to limit this to merge instead?

Reply:
Ah, so its the merges! I couldn't figure out how (as the yaml existed) codcov got a bunch of the coverage for the slow tests since I noticed we didn't run the slow ones on a schedule, but we run the slow tests whenever it isn't a pull_request. Should I do `not pr & not schedule` to match the logic of when we run slow tests? Like if someone make a commit to main directly, I don't think that would be a merge event

Reply:
@IAlibay Let me know if the updated logic is better now, I went with "not schedule && not pull request" so we can cover any case where a slow test is ran (not pull) but only upload if it isn't a schedule run.